src/components/molecules/Common/AssetModal/AssetContainer/index.tsx

```diff
@@ -22,27 +22,35 @@ export type Asset = AssetType;
 
 export type AssetSortType = SortType;
 
+export type OtherProps = {
+  teamId?: string;
+  initialAssetUrl?: string | null;
+  allowDeletion?: boolean;
+};
+
 export type Props = {
   className?: string;
   assets?: Asset[];
-  isMultipleSelectable?: boolean;
-  accept?: string;
   initialAsset?: Asset;
   selectedAssets?: Asset[];
+  isLoading?: boolean;
+  isMultipleSelectable?: boolean;
+  accept?: string;
   fileType?: "image" | "video" | "file";
   height?: number;
   hasMoreAssets?: boolean;
-  isLoading?: boolean;
   sort?: { type?: AssetSortType | null; reverse?: boolean };
   searchTerm?: string;
   smallCardOnly?: boolean;
-  onCreateAsset?: (files: FileList) => void;
+  onCreateAssets?: (files: FileList) => void;
   onRemove?: (assetIds: string[]) => void;
   onGetMore?: () => void;
-  selectAsset?: (assets: Asset[]) => void;
+  selectAssetUrl?: (asset?: string) => void;
+  onSelect?: (asset?: Asset) => void;
   onSortChange?: (type?: string, reverse?: boolean) => void;
   onSearch?: (term?: string) => void;
-};
+  handleShowURL?: (assets?: Asset[]) => void;
```

Comment:
```suggestion
  onURLShow?: (assets?: Asset[]) => void;
```

handle~ can be used for only local variables.